contracts/ParsingLibrary.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity ^0.4.24;
```

Comment:
I think this should probably be called something more descriptive, like `DNSClaimChecker`.